dev/release/rat_exclude_files.txt

```diff
@@ -52,6 +52,16 @@ python/MANIFEST.in
 python/pyarrow/includes/__init__.pxd
 python/pyarrow/tests/__init__.py
 python/requirements.txt
+r/DESCRIPTION
+r/NAMESPACE
+r/.Rbuildignore
+r/arrow.Rproj
+r/README.Rmd
+r/README.md
```

Comment:
Markdown files support license headers as comments, so there is no need to exclude them

Reply:
Thanks. ✅